wallet/src/types.rs

```diff
@@ -172,3 +172,14 @@ pub struct ExtendedKeyedSignature {
 
 #[derive(Clone, Debug, Deserialize)]
 pub struct ChainEntry(pub u32, pub String);
+
+/// Format of the output of getTransaction
+#[derive(Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct GetTransactionOutput {
```

Comment:
This is a very bad name because a transaction has inputs and outputs, and this struct does not return the transaction outputs. I'm fine with `GetTransactionResult` or `GetTransactionResponse`
```suggestion
pub struct GetTransactionResult {
```

Reply:
I agree :smile: